app/models/create_expenses_with_receipt.rb

```diff
@@ -14,11 +14,48 @@ def create
     save
   end
 
+  def update(expense)
```

Comment:
It should either be somewhere else, or rename the class. In general I prefer more classes. Then you can figure out a way of sharing common functionality.